gradle.properties

```diff
@@ -1,6 +1,17 @@
+# Disable buildFeatures flags by default
+android.defaults.buildfeatures.aidl=false
+android.defaults.buildfeatures.buildconfig=false
+android.defaults.buildfeatures.renderscript=false
+android.defaults.buildfeatures.resvalues=false
+android.defaults.buildfeatures.shaders=false
+
 android.useAndroidX=true
 android.enableJetifier=false
 android.nonTransitiveRClass=true
-org.gradle.jvmargs=-Xmx1400m
+
+# https://blog.jetbrains.com/kotlin/2022/07/a-new-approach-to-incremental-compilation-in-kotlin/
+kotlin.incremental.useClasspathSnapshot=true
+
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
```

Comment:
It looks like this commit mixes unrelated changes. Please don't do that.

Reply:
31ddac19c312266521636e1f4f7a8dc1f0cf01f7